spec/integ/matrix-to-irc.spec.js

```diff
@@ -434,6 +434,57 @@ describe("Matrix-to-IRC message bridging", function() {
         });
     });
 
+    it("should bridge matrix replies to ghosts with their nick", function(done) {
+        // Trigger an original event
+        env.mockAppService._trigger("type:m.room.message", {
+            content: {
+                body: "This is the real message",
+                msgtype: "m.text"
+            },
+            user_id: testUser.id,
+            room_id: roomMapping.roomId,
+            sender: "@irc.example_WibbleWob:some.home.server",
+            event_id: "$original:bar.com",
+            type: "m.room.message"
+        }).then(() => {
+            env.ircMock._whenClient(roomMapping.server, testUser.nick, "say",
+            function(client, channel, text) {
+                expect(client.nick).toEqual(testUser.nick);
+                expect(client.addr).toEqual(roomMapping.server);
+                expect(channel).toEqual(roomMapping.channel);
+                console.log(testUser.nick);
+                console.log(testUser.nick);
+                console.log(testUser.nick);
+                console.log(testUser.nick);
+                console.log(testUser.nick);
+                console.log(testUser.nick);
```

Comment:
6 times?

Reply:
Possibly some frantic testing made it in...